code/modules/mob/living/carbon/brain/brain_item.dm

```diff
@@ -54,18 +54,21 @@
 	else
 		user << "This one seems particularly lifeless. Perhaps it will regain some of its luster later.."
 
-/obj/item/organ/internal/brain/removed(var/mob/living/user)
+/obj/item/organ/internal/brain/removed(mob/living/user, mob/living/carbon/human/owner_)
 
-	name = "[owner.real_name]'s brain"
+	if(!owner_)
+		return ..()
 
-	var/mob/living/simple_animal/borer/borer = owner.has_brain_worms()
+	name = "[owner_.real_name]'s brain"
+
+	var/mob/living/simple_animal/borer/borer = owner_.has_brain_worms()
 
 	if(borer)
 		borer.detatch() //Should remove borer if the brain is removed - RR
 
 	var/obj/item/organ/internal/brain/B = src
-	if(istype(B) && istype(owner))
```

Comment:
`owner` is organ var. Why it have been replaced with proc argument (`owner_`)?

Reply:
Because when head is seperated from the body, it loses it's owner var(as well as the brain). However this entire code is made for the seperated head, which just contradicts itself.

Reply:
<code>var/mob/living/carbon/human/victim = owner_ ? owner_ : owner // Used to keep and put into other's removed procs.</code>

Reply:
Also. 
`var/mob/living/carbon/human/victim = owner_ ? owner_ : owner`
would not affect `owner_` in `.../organ/external/head/remove()`.

Reply:
It would, <code>if(internal_organs)
		for(var/obj/item/organ/organ in internal_organs)
			organ.removed(owner_ = victim)</code>